samples/c/get_setup_mode.c

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: LGPL-2.1-only
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates
+ *
+ * Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+ *
+ * Description: This file contains the sample code to demonstrate usage of
+ * 		cgroup_setup_mode() API.
+ */
+
+#include "../src/libcgroup-internal.h"
```

Comment:
I wouldn't expect an external program to need to include `libcgroup-internal.h`.  Shouldn't `#include <libcgroup.h>` be sufficient?

Reply:
@drakenclimber Thank you so much for the review.  Will re-spin v2 addressing all comments.